books/free-programming-books-langs.md

```diff
@@ -2248,11 +2248,12 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Easy Rust](https://dhghomon.github.io/easy_rust/) - David McLeod (HTML, PDF)
 * [From JavaScript to Rust ebook](https://github.com/wasmflow/node-to-rust/raw/HEAD/from-javascript-to-rust.pdf) - Jarrod Overson (PDF)
 * [Guide to Rustc Development](https://rustc-dev-guide.rust-lang.org)
-* [Learn Rust in Y minutes](https://learnxinyminutes.com/docs/rust/)
+* [Learn Rust in Y minutes](https://learnxinyminutes.com/docs/rust)
```

Comment:
```suggestion
* [Learn Rust in Y minutes](https://learnxinyminutes.com/docs/rust/)
```
trailing slash only needed for website root

Reply:
ooh sorry i didnt knew